docs/docusaurus/versioned_docs/version-1.4.0/lte/upgrade_140.md

```diff
@@ -0,0 +1,52 @@
+---
+id: agw_140_upgrade
+title: Upgrading from 1.3
+hide_title: true
+---
+# Upgrading from 1.3
+
+You can upgrade your access gateways remotely from the NMS or SSH directly
+into them and run an `apt-get install`.
+
+## NMS Autoupgrade
+
+If you've set up your Access Gateways in upgrade tiers already, you can upgrade
```

Comment:
NMS autoupgrade doesn't change sources. so I don't think that will work, unless someone changed the nms autoupgrade code